src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -183,6 +183,19 @@ export const getSpecialCurrencyInfo = (pluginId: string): SpecialCurrencyInfo =>
 export const SPECIAL_CURRENCY_INFO: {
   [pluginId: string]: SpecialCurrencyInfo
 } = {
+  abstract: {
+    allowZeroTx: true,
+    chainCode: 'ETH',
```

Comment:
This is used for FIO. Does FIO support Abstract? If so, is this the code they use?

Reply:
I don't know if FIO supports Abstract or not. I copied the chainCode from all the other EVM based `SpecialCurrencyInfo`s. Do these other configs need fixing too?

Reply:
Most likely you have found a bug!